frontend/src/pages/RegionalDashboard/index.js

```diff
@@ -16,16 +16,20 @@ import ReasonList from '../../widgets/ReasonList';
 import TotalHrsAndGrantee from '../../widgets/TotalHrsAndGranteeGraph';
 import './index.css';
 
-function Dashboard({ user }) {
-  const [appliedRegion, updateAppliedRegion] = useState(0);
-  const [regions, updateRegions] = useState([]);
-  const [regionsFetched, updateRegionsFetched] = useState(false);
+export default function RegionalDashboard({ user }) {
+  const hasCentralOffice = user && user.homeRegionId && user.homeRegionId === 14;
```

Comment:
It's early days for me and this codebase, but `14` stands out to me as a magic number here--a future refactor should consider introducing a helper function to tell whether a region has a central office or possibly creating a `regions.json` file for tracking these sorts of attributes.